src/bank.rs

```diff
@@ -35,10 +35,11 @@ pub const MAX_ENTRY_IDS: usize = 1024 * 16;
 pub const VERIFY_BLOCK_SIZE: usize = 16;
 
 /// Reasons a transaction might be rejected.
+#[cfg_attr(feature = "cargo-clippy", allow(large-enum-variant))]
```

Comment:
Not a big fan of doing this. I'll make it a Box<>, once all other tests succeed.